extensions/adobe/experience/campaign/orchestration/reportingevent.schema.json

```diff
@@ -109,34 +80,14 @@
         },
         "https://ns.adobe.com/experience/campaign/journey": {
           "title": "Journey for users",
-          "type": "object",
           "description": "Journey created by marketer for its users",
-          "properties": {
-            "@id": {
-              "title": "Journey unique identifier",
-              "type": "string",
-              "format": "uri",
-              "description":
-                "The unique identifier of the journey created by the marketer."
-            }
-          },
-          "required": ["@id"]
+          "$ref": "https://ns.adobe.com/experience/campaign/orchestration/experienceevent#/definitions/journey"
```

Comment:
I see my proposal to "merge" `journey` and `journeyVersion` into a single object might conflict with this definition here. Could this be applied or would that be a breaking change ?

Reply:
It would definitely be a breaking change.